ansible/playbook/tasks/roles/pmm-dump/tasks/main.yml

```diff
@@ -0,0 +1,33 @@
+---
+# Install pmm-dump, always overwrite the version before with latest
+
+- name: Download pmm-dump binary (tech preview)
+  get_url:
+    url: https://github.com/percona/pmm-dump/releases/download/0.5.1-beta/pmm-transferer
+    dest: /tmp/pmm-transferer
```

Comment:
Why we can't save file right to `/usr/local/bin/pmm-dump`?

Reply:
Sure, will do